chat-plugins/info.js

```diff
@@ -98,7 +98,7 @@ var commands = exports.commands = {
 	host: function (target, room, user, connection, cmd) {
 		if (!this.can('rangeban')) return;
 		var self = this;
-		Dnsbl.reverse(target, function(err, hosts) {
+		Dnsbl.reverse(target, function (err, hosts) {
```

Comment:
Zarel already took care of this in https://github.com/Zarel/Pokemon-Showdown/compare/bcd48fe8db5b...2bd4057e15b2